apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/CareerTimelineSection/CareerTimelineSection.tsx

```diff
@@ -36,15 +33,6 @@ const CareerTimelineSection = ({ experiences }: CareerTimelineSectionProps) => {
 
   return (
     <>
-      {features.recordOfDecision && (
-        <Heading Icon={UserCircleIcon} color="tertiary">
-          {intl.formatMessage({
-            defaultMessage: "Career timeline",
-            id: "5FK/4R",
-            description: "Heading for the users experiences",
-          })}
-        </Heading>
-      )}
```

Comment:
Looks like the opposite happened :sweat_smile:

Reply:
Whoops.  1c87e20e2e86f4626934368f908bbf7d812c85a2